src/main/java/racingcar/Car.java

```diff
@@ -9,5 +9,33 @@ public Car(String name) {
         this.name = name;
     }
 
-    // 추가 기능 구현
+    public void IncreasePosition() {
```

Comment:
IncreasePosition 메소드명이 camel case를 지키지 않고 있습니다!
```suggestion
    public void increasePosition() {
```